aspnetcore/blazor/index.md

```diff
@@ -4,120 +4,142 @@ author: guardrex
 description: Explore ASP.NET Core Blazor, a way to build interactive client-side web UI with .NET in an ASP.NET Core app.
 monikerRange: '>= aspnetcore-3.0'
 ms.author: riande
-ms.custom: seoapril2019
-ms.date: 04/18/2019
+ms.custom: "mvc, seoapril2019"
+ms.date: 04/22/2019
 uid: blazor/index
 ---
 # Introduction to Blazor
 
 By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.com/guardrex)
 
-Welcome to Blazor!
+*Welcome to Blazor!*
 
 Build interactive client-side web UI with .NET:
 
-* Build rich interactive UIs using C# instead of JavaScript.
+* Create rich interactive UIs using C# instead of JavaScript.
 * Share server-side and client-side app logic written with .NET.
 * Render the UI as HTML and CSS for wide browser support, including mobile browsers.
 
-Blazor supports core scenarios required by most apps:
+Blazor supports app development scenarios required by most apps:
```

Comment:
This Blazor feature list feels a bit awkward here. Some of these features are self-explanatory, but many of them are not.

I think it's important that this first section covers: 1. What is Blazor? and 2. Why should I care?

I think we have (1) covered, but I think we could do a better job with (2). We have some text that talks about the advantage of using Blazor in the client-side section below. How about we move it up here in place of this feature list?